registration-processor/post-processor/registration-processor-printing-stage/pom.xml

```diff
@@ -109,6 +109,11 @@
 			<artifactId>registration-processor-info-storage-service</artifactId>
 			<version>${packet.info.storage.service.version}</version>
 		</dependency>
+		<dependency>
```

Comment:
Do we need to add this Dependency?

Reply:
Without this dependency, application is throwing NoClassFoundException: io.vertx.core.Promise